response_operations_ui/templates/respondent-search/_inline-form.html

```diff
@@ -0,0 +1,23 @@
+<form action="{{ url_for('respondent_bp.search_redirect') }}" method="post" class="u-mt-l">
```

Comment:
Why do some of the files in here start with underscore?  I can't find another example of that anywhere else

Reply:
Templates that are parts of a page, rather than a page itself begin with an underscore in this.  Didn't realise that that convention hasn't been followed elsewhere.

Can convert them back if need be?

Reply:
I think changing them is best. If we find they're confusing or whatever, then we can fix that how we want down the line.  It looks like the collection_exercise section of the templates has done the same in putting all the bits in the same place as the main template.  I think it's better to be consistent than right for this.

Reply:
Yeah all done, easy enough with autorefactor.